config/migrations/2024/20241028084700-expose-verkiezingen-to-ocmw/20241028084700-copy-verkiezingen.sparql

```diff
@@ -0,0 +1,26 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX lmb: <http://lblod.data.gift/vocabularies/lmb/>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX org: <http://www.w3.org/ns/org#>
+INSERT {
+  GRAPH ?ocmwG {
+    ?verkiezing mandaat:steltSamen ?gemeenteOrgInTijd.
+    ?verkiezing ext:copiedForOCMW ?ocmw.
+    ?verkiezing mandaat:steltSamen ?ocmwOrgInTijd.
+  }
+} WHERE {
+  ?gemeente a besluit:Bestuurseenheid .
+  ?ocmw a besluit:Bestuurseenheid .
```

Comment:
You never specify which bestuurseenheid is which one. The only way it is derived is for the ocmw with the classificatiecode for the bestuursorgaan. The gemeente could also be ocmw for all we know. Which feels a bit scary.

Reply:
It might be implied because it has a `mandaat:steltSamen` relation, but still, feels a bit scary.